config/configClient.go

```diff
@@ -78,13 +124,23 @@ func (cs ConfiguredService) GetRequiredCredentialTypes(scope string) []string {
 	return types
 }
 
-func (cs ConfiguredService) GetCredentials(scope string) []Credential {
+func (cs ConfiguredService) GetScope(scope string) ScopeEntry {
 	if scope != SERVICE_DEFAULT_SCOPE {
 		return cs.ServiceScopes[scope]
 	}
 	return cs.ServiceScopes[cs.DefaultOidcScope]
 }
 
+func (cs ConfiguredService) GetCredentials(scope string) []Credential {
+	return cs.GetScope(scope).Credentials
+}
+
+func (cs ConfiguredService) GetPresentationDefinition(scope string) PresentationDefinition {
+	logging.Log().Warnf("The ScopeEntry %s", logging.PrettyPrintObject(cs.GetScope(scope)))
```

Comment:
I know its highly unlikely but what if the config from the CCS get updated between these two calls and we return a  different config than we log here

Reply:
This was a left over to be removed:)